### PR TITLE
DO NOT MERGE YET - basic shell completion

### PR DESCRIPTION
**Pull request by doniks**

### CODE REVIEW

**Comment by NeoTheThird** [comment]
If you want, I can add some helper functions in the python script to print for example the github repos, active PRs and available PPAs. For remove, you should already be able to use `ubports-qa list`.

**Comment by doniks** [comment]
@neothethird yes ,but one step at a time

1. Im stuck getting the branch to install. Will try again and ping you on tg if i need help
2. Need to master the debian packaging
3. I have a draft for remove which breaks in confusingly strange ways
4. Then I can think of more features

**Comment by NeoTheThird** [comment]
Ok, no pressure :) The convenient thing is that everybody who uses this script will automatically have the latest version, so we don't need to rush it into the stable release :P

**Comment by doniks** [comment]
> I can add some helper functions in the python script to print for example the github repos, active PRs and available PPAs. 

It would be helpful for users of ubports-qa anyway. (Such as me, still struggling to get it to work :-P ) Regardless of whether / how quickly it can be added to bash completion

File: debian/install [hunk 1]
```diff
@@ -1 +1,2 @@
 ubports-qa /usr/bin
+tools/completions/ubports-qa usr/share/bash-completion/completions/ubports-qa
```
Comment (JBBgameich) [inline debian/install]: Remove the filename `ubports-qa` from the target path `usr/share/bash-completion/completions/ubports-qa`. I think it should work then ...

**Comment by doniks** [comment]
closing because I want to test ubports-qa with a branch WITHOUT a pr

**Pull request closed without merging**